api/test/integration/test_experimental_endpoints.tavern.yaml

```
@@ -1,9 +1,87 @@
 ---
-test_name: DELETE /experimental/syscheck
+test_name: DELETE /experimental/rootcheck
 
 marks:
   - base_tests
 
+stages:
+
+  # DELETE /experimental/rootcheck
+  - name: Try to delete rootcheck scans in agent 001
```

Use `Try to` only when no data is expected in the response (permission error, denied permissions but user unaware, etc), so it is easier to identify which should be the output.
 ```suggestion
  - name: Delete rootcheck scans in agent 001
```

---

Done